apps/ui/src/apis/rides/join-ride-request.ts

```diff
@@ -0,0 +1,58 @@
+import { Message } from '@communecar/types';
+import { graphqlRequest } from '../graphql';
+
+const getUpdateQuery = (
+  userId: number,
+  rideId: number,
+  status: string,
+): string => {
+  return `mutation{
+        updateUserRideByUserIdAndRideId(input: {
+            userId: ${userId},
+            rideId: ${rideId},
+            userRidePatch: {
+              status: "${status}"
+            }
+          }) {
+            userRide {
+              userId
+              rideId
+              status
+            }
+          }
+    }`;
+};
```

Comment:
Consider moving it to a separate file so that I can also use it to change the status in the context of a request to join